tests/grid/grid_generator_subdivided_hyper_cube_01.cc

```diff
@@ -0,0 +1,53 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 - 2019 by the deal.II authors
```

Comment:
Oh, the point here was to update the first year this file (or its content) was part of the repository. We update the second year automatically.
```suggestion
// Copyright (C) 2019 by the deal.II authors
```